setup.cfg

```diff
@@ -1,5 +1,13 @@
 [flake8]
 max-line-length = 120
+exclude = alembic/versions/*
+ignore =
+    E20,   # Extra space in brackets
+    W503,  # line break before binary operator
+    W504,  # line break after binary operator
```

Comment:
Why add these?

Reply:
Disagreement between flake8 and `black` on some fortmatting stuff. Dask core also has these.